packages/calcite-components/src/components/block/block.tsx

```diff
@@ -364,8 +435,16 @@ export class Block
             title={toggleLabel}
           >
             {headerContent}
-            <calcite-icon aria-hidden="true" class={CSS.toggleIcon} icon={collapseIcon} scale="s" />
+            <div class={CSS.iconEndContainer}>
+              {iconEndEl}
+              <calcite-icon class={CSS.toggleIcon} icon={collapseIcon} scale="s" />
```

Comment:
toggleIcon is missing from CSS resources?

Reply:
It's there I believe.

Reply:
Could you review these one more time @alisonailea, in case I missed something?